README.md

```diff
@@ -182,6 +182,16 @@ open a terminal and run:
 $ grunt watch
 ```
 
+*Alternatively if you don't want to run the full watch task there are two available sub-tasks*
```

Comment:
Might include the ending colon like the sentence before this one. Also, I'm no grammar wiz, but I think you're missing a comma:
`Alternatively, if you don't want to run the full watch task there are two available sub-tasks:`